dotnet/src/Planners/Planners.Core/Handlebars/skPrompt.handlebars

```diff
@@ -2,19 +2,23 @@
 Explain how to achieve the user's goal with the available helpers with a Handlebars template.
```

Comment:
The naming of this file can be improved, instead of `skPrompt` use a name that describes the purpose of the prompt. We won't to avoid having multiple files with the same name.

Reply:
Changed to `CreatePlanPrompt.handlebars`